### PR TITLE
Update registrations

### PR DESCRIPTION
**Pull request by feclare**
Client notification is not done yet.

### CODE REVIEW

File: distro/server.go [hunk 1]
```diff
@@ -20,11 +20,19 @@ func replyPing(w http.ResponseWriter, r *http.Request) {
...
+	io.WriteString(w, "")
+	RefreshRegistrations()
+}
+
 // HTTPServer function
 func HTTPServer() http.Handler {
 	mux := bone.New()
 
 	mux.Get("/api/v1/ping", http.HandlerFunc(replyPing))
+	mux.Get("/api/v1/updateregistrations", http.HandlerFunc(replyUpdateRegistrations))
```
Comment (drasko) [inline distro/server.go]: `updateregistrations` is a bit unreadable. Please improve redability by choosing some other name for the endpoint. Some suggestions: `reg_update`, `update_registrations`, `notify_reg` or maybe even `notify/registration`, as there might be potentially some other notifications (not sure about this).

Comment (feclare) [reply distro/server.go]: I choose notify/registrations

**Comment by feclare** [comment]
Added client notifying distro when registrations change.

**Comment by drasko** [comment]
Merged, thanks!

**Pull request merged**